apps/demos/Demos/CardView/SimpleArray/jQuery/index.js

```diff
@@ -0,0 +1,7 @@
+$(() => {
+  $('#gridContainer').dxCardView({
```

Comment:
Let's replace this id with ```card-view``` or ```cardView``` (but first is better)
```suggestion
  $('#card-view').dxCardView({
```